src/_guides/language/sound-problems.md

```diff
@@ -20,7 +20,28 @@ and see [these other resources](/guides/language/type-system#other-resources).
 
 ## Troubleshooting
 
-### No type errors {#no-type-errors}
+Dart enforces a sound type system. Roughly, this means
+you can't get into a situation where the value stored in a variable is
+different from the variable's static type. Like most modern statically
+typed languages, Dart accomplishes this with a combination of
+[static (compile-time)](#static-errors-and-warnings) and [dynamic (runtime)](#runtime-errors) checking.
```

Comment:
**suggestion:** This could be trimmed down.
```suggestion
Dart enforces a sound type system. In general terms, this means you can't write code where a variable's value differs from its static type. An `int` type can't store a number with a decimal place. Dart checks variable values against their types at [compile-time](#static-errors-and-warnings) and [runtime](#runtime-errors).
```

These changes drop readability from Grade 11 to Grade 5. Sure, it's eight more words, but they're small.

Reply:
As this PR didn't write this text and there are some particularities of these adjustments that may need further review, perhaps best to leave these changes for a separate PR. 

The whole article could likely use some readability improvements, so perhaps that could be done in a follow-up PR?